client/src/components/MovieDetails.jsx

```diff
@@ -0,0 +1,24 @@
+import '../static/inline.css';
+import '../static/margin-left.css';
+
+import React from 'react';
+
+class MovieDetails extends React.Component {
+    render(){
+        return(
+            <div>
+                <div className='inline'><img src={this.props.selectedMovie.poster_path}/></div>
```

Comment:
_trivial_

For the consistency purposes, it is nice to have quotes defined in one way - actually, it doesn't matter either it would be `"double"` or `'single'` quotes.
I consider to not thinking about which kind of quote you should put while coding, but you can configure your IDE, Text Editor to get the job done automatically, set up ESLint/Prettier, etc.